web/src/components/Upload/Upload/Upload.tsx

```diff
@@ -43,6 +43,12 @@ const Upload = ({ name, multiple, handleUpload }: IUploadProps) => {
       cloudName: 'dl5elpdjy',
       uploadPreset: 'bcfnswai',
       multiple: multiple ? multiple : true,
+      folder:
+        name === 'profilePicture'
+          ? 'Profile pictures'
+          : name === 'coverImage'
+          ? 'Cover images'
+          : '',
```

Comment:
Perhaps you could extract this to a separate function like `getFolderName(name) { ... }`